src/analysis/modals/GalaxyModal.js

```diff
@@ -113,7 +113,8 @@ export const GalaxyModalBase = withDisplayName('GalaxyModal')(
       const deleteButton = h(ButtonOutline, { disabled: false, style: { marginRight: 'auto' }, onClick: () => setViewMode('deleteWarn') }, [
         'Delete Environment',
       ]);
-      const pauseButton = h(ButtonSecondary, { disabled: false, style: { marginRight: '1rem' }, onClick: pauseGalaxy }, ['Pause']);
+      // TODO: Reenable pause button once https://broadworkbench.atlassian.net/browse/PROD-905 is resolved
+      const pauseButton = h(ButtonSecondary, { disabled: true, style: { marginRight: '1rem' }, onClick: pauseGalaxy }, ['Pause']);
```

Comment:
really feels like these modals should be templated (cromwell/galaxy) as opposed to hardcoding options when we have a config for each that says whether pause is enabled/disabled... 

Oh well, not even close to relevant for the scope of this

Reply:
Would you happen to be aware of an example to follow?

Reply:
Its more of a desired end state. I think over time as we split more logic out of these modals into sub components (like Disk Options), they will become more generalized. This here is a good example of something that probably applies to all apps/runtimes (i.e. a 'action buttons' component that transforms the environment and its state into a set of valid actions)